docs/testing-components/test-driven-development/index.md

```diff
@@ -88,37 +85,33 @@ solution.
 We use `Jest` for our unit testing. While there are quite a few comparisons it can help to stick to `.toBe()` and `.not.toBe()`.  These methods come after
 the `expect()` call.
 
-We use Enzyme to render our components for testing. Enzyme can render a component in one of three different ways.  Each
-has its benefits and drawbacks.  A summary of the methods follows.
+We use React Testing Library to render our components for testing.
```

Comment:
Here we could add some info that react testing library is built do create tests based on the user perspective